docs/reference/search/request/sort.asciidoc

```diff
@@ -213,7 +213,7 @@ then Elasticsearch will handle it as if there was a mapping of type
 [[geo-sorting]]
 ==== Geo Distance Sorting
 
-Allow to sort by `_geo_distance`. Here is an example:
+Allow to sort by `_geo_distance`. Here is an example, assuming `pin.location` is a simple object storing the location of the item (otherwise `nested_path` should be set as well):
```

Comment:
mentioning `nested_path` here is confusing and unrelated.  I'd just say:

    assuming `pin.location` is a field of type `geo_point`: